integration/plugin/mountpoint/main_test.go

```diff
@@ -0,0 +1,33 @@
+package mountpoint
+
+import (
+	"fmt"
+	"os"
+	"testing"
+
+	"github.com/docker/docker/internal/test/environment"
+	"github.com/docker/docker/pkg/sysinfo"
+)
+
+var testEnv *environment.Execution
+var sysInfo *sysinfo.SysInfo
+
+const dockerdBinary = "dockerd"
+
+func TestMain(m *testing.M) {
+	var err error
+	testEnv, err = environment.New()
+	if err != nil {
+		fmt.Println(err)
+		os.Exit(1)
+	}
+
+	sysInfo = sysinfo.New(true)
+
+	testEnv.Print()
+	setupSuite()
```

Comment:
This is going to break windows CI (I just realized that these suites aren't running on windows CI but that should be fixed shortly).

It looks like `mountpoint_windows_test.go` is missing which needs to define these.